service-level-sidecar/report.js

```diff
@@ -43,34 +48,34 @@ class MetricReport {
 
 
 /**
- *
+ * @param {Array<Number>} logIds - raft session ids to retrieve metrics for (only contains info if scuba backend is enabled)
```

Comment:
can we call it sessionIds then to avoid confusion as the logger is already called `log`